generators/bdk/python/templates/bot-app/main.py.ejs

```diff
@@ -32,16 +30,19 @@ class RealTimeEventListenerImpl(RealTimeEventListener):
     stream_id = event.message.stream.stream_id
     display_name = initiator.user.display_name
     await self._bdk.messages().send_message(
-        stream_id,
-        f"<messageML>Hello {display_name}, hope you are doing well!</messageML>"
+      stream_id,
+      f"<messageML>Hello {display_name}, hope you are doing well!</messageML>"
     )
 
 
+config = BdkConfigLoader.load_from_file(os.path.join(current_dir, '..', 'resources/config.yaml'))
 logging.config.fileConfig(os.path.join(current_dir, '..', 'logging.conf'),
                           disable_existing_loggers=False)
 
 try:
   logging.info("Running bot application...")
+  if os.name == "nt" and config.proxy is not None:
```

Comment:
Could we document that in the FAQ part ?

Reply:
@symphony-elias you mean like adding a section in the known issues here: https://github.com/SymphonyPlatformSolutions/generator-symphony/blob/master/README.md ?

Reply:
I meant here: https://github.com/SymphonyPlatformSolutions/symphony-api-client-python/blob/2.0/docsrc/markdown/faq.md